pymongo/operations.py

```diff
@@ -501,6 +502,451 @@ def _add_to_bulk(self, bulkobj: _AgnosticBulk) -> None:
         )
 
 
+class ClientInsertOne(Generic[_DocumentType]):
```

Comment:
All of these Model classes look identical to the existing collection-level bulk write API ones except for having a namespace. In the interest of code consolidation, how about adding an optional `namespace` to the existing classes and a new `_add_to_client_bulk` method?